src/admin/vote/Add.tsx

```
@@ -0,0 +1,356 @@
+            >
+              <mdui-menu-item value="null">Keine</mdui-menu-item>
+              {options.map((o) => (
+                <mdui-menu-item
+                  onClick={() => {
+                    const newSelected = selected.slice();
+                    newSelected[i] = o.id;
+                    setSelected(newSelected);
+                  }}
+                  value={o.id}
+                >
+                  {o.title}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)